engine/src/engine.ts

```diff
@@ -361,26 +363,23 @@ export default class Engine {
   addPlayer(player: Player) {
     this.players.push(player);
 
-    player.data.on(`gain-${Resource.TechTile}`, () => this.processNextMove(SubPhase.ChooseTechTile));
-    player.data.on(`gain-${Resource.TemporaryStep}`, () => this.processNextMove(SubPhase.BuildMine));
+    player.data.on(`gain-${Resource.TechTile}`, () => this.processNextMove(SubPhase.ChooseTechTile, null, true));
```

Comment:
Would this make building a temple illegal when no tech tiles are available? Because it should only be illegal when taking Itar's action or the QIC action

Reply:
good catch!